pkg/codegen/docs/gen.go

```diff
@@ -606,7 +588,7 @@ func (mod *modContext) genConstructorTS(r *schema.Resource, argsOptional bool) [
 			OptionalFlag: argsFlag,
 			Type: propertyType{
 				Name: argsType,
-				Link: argsDocLink,
+				Link: "#input",
```

Comment:
For TS, C#, and Go, the "args" type will now just link to the "Input Properties"  section on the same page which details the properties of the args type.